index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fikerte Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/1652c6c81e.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+        <a href="#"><span>My Logo</span></a>
+        <a href="#"><i class="fas fa-bars"></i></a>
+    </header>
+    <section class="headline">
+        <h2>I'm Fikerte Glad to see you!</h2>
+        <p>I’m a software developer! I can help you build a product , 
+           feature or website Look through some of my work and experience!
+           If you like what you see and have a project you need coded, 
+           don’t hestiate to contact me.</p>
+        <a href="#">Let's connect</a>
+        <section class="social-icons">
+            <a href="#"><i class="fab fa-twitter"></i></a>
+            <a href="#"><i class="fab fa-linkedin-in"></i></a>
+            <a href="#"><i class="fab fa-medium-m"></i></a>
+            <a href="#"><i class="fab fa-github"></i></a>
+            <a href="#"><i class="fab fa-angellist"></i></a>
+        </section>
```

Comment:
- [ ] The list of icons has a `ul` tag (instead of `section`) to group them, and `li` per icon; each icon has a link (`a` tag). Please change it.